### ISSUE TITLE
Blog post recipe doesn't have images in media

### ISSUE THREAD

**Issue by sebastienros** [OrchardCMS/OrchardCore#6299]
Gives a bad experience when we try to create new posts as we have to provide a background image. The field is mandatory. Or also make the field not mandatory so it can fallback to the default background.

**Comment by agriffard** [comment]
You experienced this when you tested on try.oc.
With the current version, there are 3 media available.

**Comment by sebastienros** [comment]
And locally it's fine too, is it just me?

**Comment by deanmarcussen** [comment]
I'd be keen to see the media field and category / tags fields optional, not required.

I'm not seeing any media in the library by default on a new install of the blog theme either

**Comment by sebastienros** [comment]
I agree that everything should be optional, in terms of experience that's harder to create a demo blog post, and even in terms of blogging, why would you force to have a tag, a category. For the background even if it was mandatory, then a default should be set. But in that case I prefer it to be optional too. Also maybe we could put the content under the title so it's easier as an editor.

**Comment by deanmarcussen** [comment]
I'm pretty sure the view was written to fall back to a default image anyway (to allow for backwards compatibility), which should would work ok as a default
